<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password</class>
 <widget class="QDialog" name="password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>231</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>智能大棚---找回密码</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(86, 118, 162);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>邮    箱:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ui_email">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
           </property>
           <property name="maxLength">
            <number>100</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="ui_verification_label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>验 证 码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ui_verification">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="ui_passwd_label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>密    码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ui_passwd">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="ui_repasswd_label">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>确认密码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ui_repasswd">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>12</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
           </property>
           <property name="maxLength">
            <number>15</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="cursorMoveStyle">
            <enum>Qt::LogicalMoveStyle</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ui_code">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
           </property>
           <property name="text">
            <string>获取验证码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
